docs/deprecated_wiki/Contributing-to-the-Code.md

```diff
@@ -1,141 +0,0 @@
-## Making a Pull Request
```

Comment:
This information is out of date and has been replaced by https://github.com/aces/Loris/blob/master/CONTRIBUTING.md which is being actively maintained.